crates/sui-framework/packages/deepbook/sources/custodian.move

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Mysten Labs, Inc.
```

Comment:
This code should no longer be used--we should delete everything that the upgrade policy will allow to avoid accidental future refactors/cleanups.

Reply:
Should we abort everywhere with "not implemented"?